raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -76,6 +76,11 @@
         "wait for writes to accumulate before flushing them to disk.";
```

Comment:
Outside the scope of this PR but how about changing this description to:
> The duration in milliseconds that the leader will wait for writes to accumulate before appending to the topic partition.

Reply:
I updated the doc for the maxUnflushedBytes config.